cmd/dashboard.go

```diff
@@ -152,9 +152,10 @@ var DashboardCmd = &cobra.Command{
 }
 
 func init() {
-	DashboardCmd.Flags().BoolVarP(&kubernetesMode, "kubernetes", "k", false, "Start Dapr dashboard in local browser")
-	DashboardCmd.Flags().BoolVarP(&dashboardVersion, "version", "v", false, "Check Dapr dashboard version")
-	DashboardCmd.Flags().IntVarP(&dashboardLocalPort, "port", "p", defaultLocalPort, "The local port on which to serve dashboard")
+	DashboardCmd.Flags().BoolVarP(&kubernetesMode, "kubernetes", "k", false, "Start proxy for Dapr dashboard on Kubernetes and opens local browser")
```

Comment:
Still do not like this one, it is too implementation specific on the proxy. See my original suggestion to change this too.

Reply:
My feedback was that the browser is not really attached to K8s.

"Opens Dapr dashboard in local browser attached to Kubernetes cluster"

Reply:
Updated, see if it reads better.